code/index.html

```diff
@@ -1,22 +1,229 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
+    <script src="https://use.fontawesome.com/bb61c0507d.js"></script>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <title>Dev News</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./assets/css/main.css" />
+    <meta name="description" content="A dev new website" />
+    <meta name="viewport" content="width=device-width,initial-scale=1.0" />
+    <script src="./app.js"></script>
   </head>
   <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
+    <!-- START HEADER -->
+    <header class="main-header">
+      <div class="main-wrapper">
+        <div class="desktop-menu-cnt">
+          <div class="header-container">
+            <!-- LOGO -->
+            <a href="/" class="logo">
+              <img src="./assets/logo.svg" alt="" />
+            </a>
+
+            <!-- MOBILE NAVBAR -->
+            <a
+              href="javascript:void(0);"
```

Comment:
You don't need that really - far better approach would be to set `href` value as `#`